src/settings.c

```diff
@@ -275,62 +282,107 @@ static int settings_format_setting(struct setting *s, char *buf, int len)
   return buflen;
 }
 
-static void settings_msg_callback(u16 sender_id, u8 len, u8 msg[], void* context)
+static void settings_write_callback(u16 sender_id, u8 len, u8 msg[], void* context)
 {
-  (void)sender_id; (void) context;
+  (void) context;
+
+  if (sender_id != 0x42)
```

Comment:
Would it be good to `log_warn` or `log_info` when receiving a message that requires sender_id 0x42 but receives it with a different sender_id?